stable/wordpress/templates/ingress.yaml

```diff
@@ -2,16 +2,19 @@
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
- name: {{ template "fullname" . }}
- labels:
+  name: {{ template "fullname" . }}
+  labels:
     app: {{ template "fullname" . }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
- annotations:
-   {{- range $key, $value := .Values.ingress.annotations }}
-     {{ $key }}: {{ $value | quote }}
-   {{- end }}
+  annotations:
+    {{- if .Values.ingress.tlsEnabled }}
```

Comment:
Is a new boolean value really needed? We could just check if the `tls` object is defined like in the latter part of this template. This will also keep the UX compatible with other charts, and not require users to remember to toggle two values when using tls.